tools/pip/doc/CPU_ADDITIONAL.md

```diff
@@ -40,3 +40,11 @@ To install, use:
 ```bash
 pip install mxnet
 ```
+
+Nightly Builds
+--------------
+The nightly builds for this package can be found at: https://dist.mxnet.io/python/cpu
+To install the latest nightly build, use:
+```bash
+pip install --pre mxnet -f https://dist.mxnet.io/python/cpu
```

Comment:
Let's use https://dist.mxnet.io/python/all here to make it easier for users to install other variants of mxnet pip package
```suggestion
pip install --pre mxnet -f https://dist.mxnet.io/python/all
```